app/models/person.rb

```diff
@@ -372,6 +389,18 @@ def address_for_letter
     Person::Address.new(self).for_letter
   end
 
+  def second_factor_required?
+    two_factor_authentication.present? || two_factor_authentication_enforced?
```

Comment:
Wie läuft die Migration ab, wenn Kunden beginnen, für einzelne Rollen die 2FA obligatorisch zu machen? Werden dann Personen mit diesen Rollen ausgeschlossen wenn sie bisher keine 2FA eingerichtet haben? Wie ist es wenn ich erst später eine Rolle dazubekomme mit der ich zu 2FA gezwungen werde?

Reply:
Also sobald sich eine Person mit einer 2FA erzwungenen Rolle versucht anzumelden, muss diese Person nach den Credentials noch das 2FA einrichten. Vorher kommt sie nicht in Hitobito.
Das 2FA wird also jeweils beim nächsten Login erzwungen.